Build/Test/cibuild.sh

```diff
@@ -70,4 +70,5 @@ fi
 
 echo "Run integration tests"
 INTEGRATION_BOOTSTRAP=".Build/vendor/nimut/testing-framework/res/Configuration/FunctionalTestsBootstrap.php"
-.Build/bin/phpunit --colors -c Build/Test/IntegrationTests.xml --bootstrap=$INTEGRATION_BOOTSTRAP --coverage-clover=coverage.integration.clover
\ No newline at end of file
+.Build/bin/phpunit --colors -c Build/Test/IntegrationTests.xml --bootstrap=$INTEGRATION_BOOTSTRAP --coverage-clover=coverage.integration.clover
```

Comment:
there is a check or ` &&` between commands needed. 
By failing of this command, the tests are still green https://travis-ci.org/github/TYPO3-Solr/ext-solr/jobs/688803625

Reply:
yes! will have a look

Reply:
I fixed this, hope that's OK as is now.